### ISSUE TITLE
Documentation is confusing in relation to flowcharts

### ISSUE THREAD

**Issue by ewann** [francoislaberge/atom-diagrams#3]
Nice project :-)

At the moment the documentation is a little confusing in the relation to flowchart support:

https://github.com/francoislaberge/atom-diagrams/blob/master/README.md

States the file extension .flowchart is supported, however:

https://github.com/francoislaberge/atom-diagrams/blob/master/lib/diagrams.js

doesn't currently have it present in the list of supported extensions. 

If anyone else lands here and is disappointed by that, consider reviewing:

http://graphviz.org/Documentation/dotguide.pdf

and / or

http://melp.nl/2013/08/flow-charts-in-code-enter-graphviz-and-the-dot-language/

For an alternative approach that works in version 0.26.0 of this Atom plugin.

**Comment by seflless** [comment]
Hey @ewann, thanks for pointing this out. Sorry for the confusion about flowchart support. I corrected the documentation to remove mention of flowchart support. It's supported in the matching [diagrams cli](https://npmjs.com/diagrams) but I didn't get it working well yet in this package.

 It's in the plans. I haven't worked on this project in a while. I'll take a quick look now and see how much work it would be, I forget why I dropped working on it, I think my experience so far is that Dot is more versatile than the underlying [flowchart.js](http://flowchart.js.org/) project, supporting nested graphs and images and more. I want to have feature parity between this and the diagrams tool, just haven't caught up to it yet.
